.gitignore

```diff
@@ -1,2 +1,3 @@
 *~
 target/
+.vscode/
```

Comment:
Adding .gitignore entries is fine, I'm just not sure it should be part of this change.

Reply:
I can remove this from the pull request